### PR TITLE
Delete L81: Custom Audience in JWT Access Credentials and Google Default Credentials

### PR DESCRIPTION
**Pull request by yihuazhang**
This feature is no longer needed.

### CODE REVIEW

File: L81-core-cpp-jwt-token-audience.md
Comment (markdroth) [inline L81-core-cpp-jwt-token-audience.md:12]: Since this is now about setting scope instead of setting audience, please also change the gRFC title and rename the file.

File: L81-core-cpp-jwt-token-audience.md
Comment (markdroth) [inline L81-core-cpp-jwt-token-audience.md:65]: `= false`

File: L81-core-cpp-jwt-token-audience.md
Comment (markdroth) [inline L81-core-cpp-jwt-token-audience.md:5]: I guess you can change this back to `Implemented`, since your PR is approved. :)

**Review by markdroth — COMMENTED** [review]
Overall, looks good.  Just a few minor comments.

File: L81-core-cpp-jwt-token-audience.md
Comment (yihuazhang) [reply L81-core-cpp-jwt-token-audience.md]: Done.

File: L81-core-cpp-jwt-token-audience.md
Comment (yihuazhang) [reply L81-core-cpp-jwt-token-audience.md]: Done.

File: L81-core-cpp-jwt-token-audience.md
Comment (yihuazhang) [reply L81-core-cpp-jwt-token-audience.md]: Done :)

**Comment by yihuazhang** [comment]
Thanks for the quick review, Mark!

**Review by markdroth — APPROVED** [review]
This looks great!  Let's wait until next week to merge this, just to give people a chance to give feedback.

**Comment by markdroth** [comment]
We've decided to abandon this gRFC, so please change this PR to simply remove it.  Thanks!

**Comment by yihuazhang** [comment]
F.Y.I. @markdroth, the gRFC is reverted now.

**Review by markdroth — APPROVED** [review]
Thanks for doing this!

**Pull request merged**
